src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/ProposedApisFeature.ts

```diff
@@ -7,26 +7,25 @@ import * as vscode from 'vscode';
 import * as vscodeapi from 'vscode';
 import { RazorDocumentManager } from './RazorDocumentManager';
 import { RazorDocumentSynchronizer } from './RazorDocumentSynchronizer';
-import { RazorLanguage } from './RazorLanguage';
 import { RazorLanguageServiceClient } from './RazorLanguageServiceClient';
 import { RazorLogger } from './RazorLogger';
-import { RazorDocumentSemanticTokensProvider } from './Semantic/RazorDocumentSemanticTokensProvider';
+
 export class ProposedApisFeature {
     constructor(
+        // @ts-ignore these aren't used for now
```

Comment:
I just realized that I'm being silly, I didn't want to change the signature, but this is only used within our repo so it's not a big problem to do so.